packages/hub/src/utils/WebBlob.spec.ts

```diff
@@ -0,0 +1,51 @@
+import { describe, expect, it } from "vitest";
+import { WebBlob } from "./WebBlob";
+
+describe("WebBlob", async () => {
+	const resourceUrl = new URL("https://huggingface.co/spaces/aschen/push-model-from-web/raw/main/mobilenet/model.json");
```

Comment:
I'm not sure about the purpose of such test since we already tested the `slice` with a file hosted on HF. Since it's only using standard HTTP protocol then I think it should works everywhere and if it doesn't then it's not our reponsability but rather a mistake in the HTTP support from the server.

If you want I can add a test but I will add it in the `commit.spec.ts` file instead since we are uploading a LFS file there

Reply:
> I'm not sure about the purpose of such test since we already tested the slice with a file hosted on HF. 

It's two different paths, for large files we do not proxy them but _redirect_ to S3 / Cloudfront
 
> I think it should works everywhere and if it doesn't then it's not our reponsability but rather a mistake in the HTTP support from the server.

I agree in an ideal world it's not the lib's responsibility but the AWS configuration / devops team, still loading large files from HF is a significant use case, and an early warning system if it stops being true is appreciated. 

From HF's perspective, what matter is that things work, and even if it's an AWS CORS misconfiguration that stops the lib from working as it should, we want to detect / bubble up the error fast.

Reply:
Understood :+1: I will add a test for a LFS file

Reply:
yea my initial comment wasn't super clear either, I meant to add a test to see if it's loaded as a lazy blob or not.

The "slice" comment was just in case we wanted to test data downloaded for the LFS file, we don't need to download the whole LFS file (since they can be several GBs) just test on a small portion